From 3d7116c1fea52e036025fc4386e5c9e4545c22ed Mon Sep 17 00:00:00 2001
From: Morgan Rhodes <morgan@puppet.com>
Date: Fri, 26 Jun 2020 09:44:42 -0700
Subject: [PATCH] (maint) Use an alternative filename for the build metadata

This will make building with non-tagged runtimes (or locally built
artifacts) possible.

This was made possible by
https://github.com/puppetlabs/vanagon/pull/651.
---
 configs/projects/puppet-agent.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/projects/puppet-agent.rb b/configs/projects/puppet-agent.rb
index cbd5ecb15..f6ab40cea 100644
--- a/configs/projects/puppet-agent.rb
+++ b/configs/projects/puppet-agent.rb
@@ -13,7 +13,7 @@
 
   settings_uri = File.join(runtime_details['location'], "#{proj.settings[:puppet_runtime_basename]}.settings.yaml")
   sha1sum_uri = "#{settings_uri}.sha1"
-  metadata_uri = File.join(runtime_details['location'], "#{runtime_details['version']}.build_metadata.agent-runtime-#{agent_branch}.#{platform.name}.json")
+  metadata_uri = File.join(runtime_details['location'], "#{proj.settings[:puppet_runtime_basename]}.json")
   proj.inherit_yaml_settings(settings_uri, sha1sum_uri, metadata_uri: metadata_uri)
 
   # (PA-678) pe-r10k versions prior to 2.5.0.0 ship gettext gems.
